ThingIFSDK/ThingIFSDK/TargetThing.swift

```diff
@@ -28,4 +28,23 @@ public extension TargetThing {
             return self.typedID.id
         }
     }
+
+}
+
+internal func makeTargetThing(
+  _ json: [String : Any],
+  layoutPosition: LayoutPosition,
+  vendorThingID: String? = nil) throws -> TargetThing
+{
+    var jsonWithVendorThingID = json
```

Comment:
Is necessary?
I guess json has already been copied when makeTargetThing is called.